docs/overview/fs.rst

```diff
@@ -183,8 +183,8 @@ The implementation is documented below. Note that the FS functions make calls
 to other driver FS functions when the cache is enabled. i.e., ``lv_fs_read`` may call the driver's ``seek``
 so the driver needs to implement more callbacks when the cache is enabled.
 
-``lv_fs_read`` :sub:`(behavior when the cache is enabled)`
--------------------------------------------------
+``lv_fs_read`` :sub:`(Behavior When Cache is Enabled)`
+----------------------------------------------------------
```

Comment:
Is this ok?

Reply:
The capitalization looks good to me :slightly_smiling_face:  I was wondering about the number of dashes.

Reply:
> The capitalization looks good to me 🙂 I was wondering about the number of dashes.

\*\*laughing at self\*\*  Yes it is okay for it to be longer, but generates a `sphinx` error when it is too short.  This too is fixed (for consistency -- made to equal the length of the title) in subsequent work (not yet submitted).